addons/website_slides/static/src/xml/website_slides_fullscreen.xml

```diff
@@ -0,0 +1,122 @@
+<templates id="template" xml:space="preserve">
+    <t t-name="website.slides.fullscreen">
+            <div t-if="slide.slide_type == 'video' &amp;&amp; !showMiniQuiz" class="player embed-responsive embed-responsive-16by9 embed-responsive-item">
+                <iframe id="youtube-player" t-att-src="embed_url" allowFullScreen="true" frameborder="0" enablejsapi="1" autoplay="1" allow="autoplay"></iframe>
+            </div>
+            <div t-if="(slide.slide_type == 'presentation' || slide.slide_type == 'document') &amp;&amp; !showMiniQuiz" class="embed-responsive embed-responsive-4by3 embed-responsive-item ">
```

Comment:
slide.slide_type in ['presentation', 'document']

Reply:
Done